src/cinnamon/sass/_common.scss

```diff
@@ -1331,11 +1331,11 @@ StScrollBar {
     transition-duration: 100;
     color: transparentize($selected_fg_color, 0.4);
 
-    &.window-list-item-box-right {
+    &.right {
     padding-left: 0px;
     padding-right: 0px;
     }
-    &.window-list-item-box-left {
+    &.left {
```

Comment:
Could ask one favor here? Please use indentation consistent with the rest of the theme.